src/main/java/no/nav/syfo/service/NaermesteLedersMoterService.java

```diff
@@ -15,21 +21,28 @@
 
     private MoteService moteService;
 
+    private PdlConsumer pdlConsumer;
+
     @Inject
     public NaermesteLedersMoterService(
             NarmesteLederConsumer narmesteLederConsumer,
-            MoteService moteService
+            MoteService moteService,
+            PdlConsumer pdlConsumer
     ) {
         this.narmesteLederConsumer = narmesteLederConsumer;
         this.moteService = moteService;
+        this.pdlConsumer = pdlConsumer;
     }
 
-    public List<Mote> hentNaermesteLedersMoter(String nlAktoerId) {
-        List<NarmesteLederRelasjon> narmesteLederRelasjoner = narmesteLederConsumer.narmestelederRelasjonerAnsatte(nlAktoerId);
+    public List<Mote> hentNaermesteLedersMoter(String lederAktorId) {
+        Fodselsnummer lederFnr = pdlConsumer.fodselsnummer(new AktorId(lederAktorId));
+
+        List<NarmesteLederRelasjonDTO> narmesteLederRelasjoner = narmesteLederConsumer.ansatteTilInnbyggerSystem(lederFnr.getValue());
 
         List<Mote> moter = new ArrayList<>();
-        for (NarmesteLederRelasjon narmesteLederRelasjon : narmesteLederRelasjoner) {
-            List<Mote> ansattesMoter = moteService.findMoterByBrukerAktoerIdOgAGOrgnummer(narmesteLederRelasjon.getAktorId(), narmesteLederRelasjon.getOrgnummer());
+        for (NarmesteLederRelasjonDTO narmesteLederRelasjon : narmesteLederRelasjoner) {
+            AktorId innbyggerAktorId = pdlConsumer.aktorId(new Fodselsnummer(narmesteLederRelasjon.getArbeidstakerPersonIdentNumber()));
```

Comment:
Hadde kanskje vært bra med caching i pdlConsumer (både for dette og for fodselsnummer-kallet ovenfor).